classes/UpgradeSelfCheck.php

```diff
@@ -32,6 +32,21 @@
 
 class UpgradeSelfCheck
 {
+    /**
+     * PHP and PrestaShop Compatibles Versions
+     */
+    const PHP_PS_VERSIONS = ['1.6.x' => ['5.2', '7.1'],
```

Comment:
```suggestion
    const PHP_PS_VERSIONS = ['1.6' => ['5.2', '7.1'],
```

1.6.x is not a version :thinking: 
I'm also not sure all 1.6 versions are compatible with 7.1